crates/bevy_render/src/render_graph/graph.rs

```diff
@@ -101,6 +103,70 @@ impl RenderGraph {
         id
     }
 
+    /// Removes the `node` with the `name` from the graph.
+    /// If the name is does not exist, nothing happens.
+    pub fn remove_node(&mut self, name: impl Into<Cow<'static, str>>) {
+        let name = name.into();
+        if let Some(id) = self.node_names.remove(&name) {
+            if let Some(node_state) = self.nodes.remove(&id) {
+                // Remove all edges from other nodes to this one. Note that as we're removing this
+                // node, we don't need to remove its input edges
+                for input_edge in node_state.edges.input_edges.iter() {
+                    match input_edge {
+                        Edge::SlotEdge {
+                            input_node: _,
+                            input_index: _,
+                            output_node,
+                            output_index: _,
```

Comment:
```suggestion
                            output_node,
                            output_index: _,
                            input_node: _,
                            input_index: _,
```
minor change, but I find it much easier to read because it makes it clear that the edge goes from the output node to the input node (same at the other locations)

Reply:
Agree with this. In some places, it is already the other way around. Should be `output` followed by `input` everywhere, for consistency.

Reply:
I really feel we should rename all of this because input and output are confusing. But sure, I can order them that way.